biojava-structure/src/main/java/org/biojava/nbio/structure/io/cif/CifStructureConsumerImpl.java

```diff
@@ -885,11 +896,26 @@ public void consumeStructConnType(StructConnType structConnType) {
 
     @Override
     public void consumeStructKeywords(StructKeywords structKeywords) {
+    	//There may be a simpler implementation or a better implementation using
+    	// Java streams. However, I am not sure about the returned value from 
+    	// getText().values()
+    	ArrayList<String> keywordsList = new ArrayList<String>();
+    	Iterator<String> iterator = structKeywords.getText().values().iterator();
+    	while (iterator.hasNext()) {
+			String longString = (String) iterator.next();
+			String[] strings = longString.split(" *, *");
+			for (String string : strings) {
+				keywordsList.add(string.trim());
+			}
+		}
+    	structure.setKeywords(keywordsList);
+
         StrColumn pdbxKeywords = structKeywords.getPdbxKeywords();
-        // TODO what is the correct format for these?
         if (pdbxKeywords.isDefined()) {
+        	//TODO I think no need for the Collectors.joining(", ") part.
```

Comment:
Yes, you are correct. This can be simplified to:
```
        StrColumn pdbxKeywords = structKeywords.getPdbxKeywords();
        if (pdbxKeywords.isDefined()) {
            String keywords = pdbxKeywords.get(0);
            pdbHeader.setClassification(keywords);
        }
```